server_addon/traypublisher/client/ayon_traypublisher/plugins/create/create_editorial.py

```diff
@@ -676,6 +676,8 @@ def _get_base_instance_data(
             "shotName": shot_name,
             "variant": variant_name,
             "task": None,
+            "newHierarchyIntegration": True,
+            # Backwards compatible
             "newAssetPublishing": True,
```

Comment:
Should we now start removing this older data key to ensure the new one does what we want it to do?

Reply:
You can have new traypublisher addon and old ayon-core addon...

Reply:
Should we add a `remove backwards compatibility by version x.y.z` label to the comment so we know when to remove it in the future?

Reply:
can you not make use of the package `ayon_required_addons` to force a minimum version?

Reply:
> Should we add a remove backwards compatibility by version x.y.z label to the comment so we know when to remove it in the future? Or do we prepare an issue now to remind us in say 3 months?

At this moment specific version is too "wild" guess. But I guess we can add at least date when it "started" to be deprecated?

> can you not make use of the package ayon_required_addons to force a minimum version?

You can, but core cannot. And at this moment we don't know about all unofficial addons using this.